extension/docs/DevelopmentGuide.md

```diff
@@ -0,0 +1,41 @@
+# Development guide
```

Comment:
for this document, I would suggest to check the notification document, because lots of things are common between these guides.. https://github.com/commercetools/commercetools-adyen-integration/pull/45/files#diff-7d9966222b5102e10716ec93fe9b5a2a

Reply:
I think @LEQADA checked my docs as he wrote his docs after me, so it should be quite similar.